handler.go

```diff
@@ -47,7 +45,6 @@ import (
 
 	// Allow building Pilosa without the web UI.
 	_ "github.com/pilosa/pilosa/statik"
```

Comment:
can this import be moved elsewhere so that Pilosa core doesn't depend directly on statik? (If go generate has been run, then I believe github.com/pilosa/pilosa/statik imports rakyll/statik/fs)